translations/cn/try_ruby_100.md

```diff
@@ -0,0 +1,15 @@
+---
+lang:   CN
+title:  停下，你在狂吠!
+answer: ^\d{1,}$
+ok:     只有字符串才能被逆置！
+error:  
+---
+
+你不可以逆置数字40，我猜你可以抱着你的显示器并把它对着镜子，但逆置一个数字听起来就不合理。
```

Comment:
A comment from Bitfox from Ruby mailing list:

```
This sounds worse. it could be better as this:

40是一个数字，数字并不能反转。虽然大千世界，如镜花水月，如梦亦如幻。但数字在编程世界里不能反转。

Thanks
```

Reply:
Emmmm, I don't think this is better. The original English version is :
`You can't reverse the number forty. I guess you can hold your monitor up to the mirror, but reversing a number just doesn't make sense.`
But he translated this to:
`40 is a number, and the number is not reversed. Although the world is as big as a mirror, like a dream or a fantasy. But numbers cannot be reversed in the programming world.`
Sounds weird to me🤔 

and did not translate this sentence：
`I guess you can hold your monitor up to the mirror`

Reply:
I will send him the original strings (I only sent him your PR), very possible some meaning has been lost in the process. You may use a part of this suggestion, or take it in the whole, or none of it, in general use your best judgement, this process is just to improve the translation :D

Tomorrow we may also have another reviewer.

Reply:
Got that

Reply:
Anyway it's all fine. The original English version was some kind of an abstract joke and those are usually enormously hard to translate.

Reply:
Indeed, yet there are very many such jokes in this website